SU2_CFD/include/numerics_structure.hpp

```diff
@@ -2110,42 +2140,85 @@ class CUpwSca_TurbSA : public CNumerics {
 };
 
 /*!
- * \class CUpwSca_TurbML
- * \brief Class for doing a scalar upwind solver for the Spalar-Allmaral turbulence model equations.
+ * \class CUpwSca_TurbSA
+ * \brief Class for doing a scalar upwind solver for the Spalar-Allmaras turbulence model equations.
  * \ingroup ConvDiscr
  * \author A. Bueno.
  * \version 5.0.0 "Raven"
  */
-class CUpwSca_TurbML : public CNumerics {
+class CUpwSca_TurbSA : public CUpwSca_Abstract {
```

Comment:
I'll make that change one of the next commits.  I had no idea what those extra classes were for.  Thanks for the clarification!